packages/strapi-admin/admin/src/containers/MarketplacePage/PluginBanner/index.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { FormattedMessage, useIntl } from 'react-intl';
 import Wrapper from './Wrapper';
 import LogoStrapi from '../../../assets/images/banner_strapi-rocket.png';
 
-const PluginBanner = () => {
+const PluginBanner = ({ emitEvent }) => {
```

Comment:
The `emitEvent` function isn't used to define the `Banner` component. IMO this component shouldn't depend on this kind of props. Can you get the `emitEvent` from the global context please.

```js
import { useGlobalContext } from 'strapi-helper-plugin';

const { emitEvent } = useGlobalContext();

emitEvent('didGoToStrapiAwesome');
```